github/templates/README.md

```diff
@@ -0,0 +1,79 @@
+PROJECT NAME
```

Comment:
I also think '#' are far easier to read & use, but won't fight on this. just wanted to leave my 2 cents :moneybag: